content/v2.0/security/https_setup.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Enabling HTTPS with InfluxDB
+description: >
+  Enable HTTPS and Transport Security Layer (TLS) secure communication between clients and your InfluxDB servers.
+weight: 201
```

Comment:
Change this weight to 101.
Also change the following weights to make this doc appear at the top of the security section:

- `content/v2.0/security/tokens/_index.md` to 102
- `content/v2.0/security/secrets/_index.md` to 103